src/init.cpp

```diff
@@ -207,6 +207,7 @@ static void CreateNewConfigFile()
         << "addnode=london.grcnode.co.uk\n"
         << "addnode=nuad.de\n"
         << "addnode=seeds.gridcoin.ifoggz-network.xyz\n"
+        << "addnode=seed.gridcoin.pl\n"
```

Comment:
```suggestion
        << "addnode=seeds.gridcoin.ifoggz-network.xyz\n"
        << "addnode=seed.gridcoin.pl\n"
```

Let's remove nuad.de as well since the hostname fails to resolve.